Mlem/API/APIClient.swift

```diff
@@ -41,6 +43,12 @@ class APIClient {
             throw APIClientError.response(apiError, statusCode)
         }
         
+        decoder.dateDecodingStrategyFormatters = [ DateFormatter.expandedTWithTimezone,
+                                                   DateFormatter.expandedT,
+                                                   DateFormatter.standardT,
+                                                   DateFormatter.standard,
+                                                   DateFormatter.yearMonthDay ]
+        
```

Comment:
I don't know if this PR will be merged quickly, but this might be useful on its own, could you separate this in its own PR?

Reply:
I have! https://github.com/buresdv/Mlem/pull/208